distribution.pm

```diff
@@ -104,7 +104,7 @@ sub script_run {
 
     testapi::type_string "$name";
     if ($wait > 0) {
-        my $str = bmwqemu::random_string(5);
+        my $str = bmwqemu::hashed_string("SR$name$wait");
```

Comment:
I agree. it's ugly but I could not find a better way. However I would have suggested to put it into a well documented helper function anyway :-)
Don't the different "script" functions have too much duplicate code anyway?